internal/providers/docker/components/container/mount_test.go

```diff
@@ -0,0 +1,54 @@
+package container
+
+import (
+	"os/user"
+	"path"
+	"testing"
+
+	"github.com/docker/docker/api/types/mount"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestMakeMount(t *testing.T) {
+	user, err := user.Current()
```

Comment:
heh, i hadn't gotten this far when i made the earlier comment about injecting the user as a parameter :)